src/style.css

```diff
@@ -1,3 +1,76 @@
-body {
+:root {
+  --blue-color: #00acdc;
+}
+
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.header-container {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-top: 10px;
+}
+
+.logo img {
+  width: 40px;
+  height: auto;
+  margin-left: 50px;
+  margin-top: 4px;
```

Comment:
The `.logo img` class uses fixed width and margin values. Consider using relative units like percentages or `em` for better responsiveness across different screen sizes.